.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,34 @@
+name: Backend Pull Request
+
+on:
+  pull_request:
+    branches:
+      - '*'
```

Comment:
Maybe we should consider not to include and execute the workflow for release branches, e.g `rel-<version>`.

Reply:
We agreeed on using tags for releases. Will mark it as resolved.